Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashC.h

```diff
@@ -93,6 +93,8 @@ void bsg_kscrash_reinstall(const char *const crashReportFilePath,
                            const char *const crashID);
 
 /** Set whether or not to print a stack trace to stdout when a crash occurs.
+ *
+ * Note: This has been removed, and now does nothing.
```

Comment:
Can we remove this function altogether?

Reply:
We could, but I was worried about it breaking someone's build since it's technically a public API.

Reply:
I don't think there's any way to import this via public Bugsnag headers (?)